tensorboard/webapp/plugins/plugins_container.ts

```diff
@@ -30,7 +30,7 @@ const activePlugin = createSelector(
   getPlugins,
   getActivePlugin,
   (plugins, id): UiPluginMetadata | null => {
-    if (!id || !plugins) return null;
+    if (!id || !plugins[id]) return null;
```

Comment:
This is not directly related to the main changes in this PR, but fixing a pre-existing bug, right?

Reply:
It is related. After this change, the store now has `activePlugin` state but no the `plugins`. Previously, the `activePlugin` was available only after `plugins` were set. 

In other words, only after this change, this bug manifests.